core/core.go

```diff
@@ -542,6 +542,16 @@ func (app *App) Start() error {
 		}
 	}()
 
+	// NOTE(jalextowle): If we are already more than 128 blocks behind, there is no need
+	// to check for missing .
```

Comment:
Can you finish this comment and elaborate on why we don't need to do this check if we're more than 128 blocks behind?